tests/swin/test_modeling_swin.py

```diff
@@ -306,6 +310,77 @@ def test_attention_outputs(self):
                     [self.model_tester.num_heads[0], window_size_squared, window_size_squared],
                 )
 
+    def test_model_outputs_equivalence(self):
```

Comment:
Any reason you need to overwrite this test?

Reply:
I don't remember, removed